packages/Discretization/src/DTK_FE.hpp

```diff
@@ -34,6 +34,8 @@
 #include <Intrepid2_HGRAD_WEDGE_C1_FEM.hpp>
 #include <Intrepid2_HGRAD_WEDGE_C2_FEM.hpp>
 
+#include <memory>
```

Comment:
Did you mean to add this include directive?

Reply:
yeah not sure why that is there.

Reply:
oh yeah he was using `std::unique_ptr`

Reply:
https://github.com/ORNL-CEES/DataTransferKit/blob/36ee35f08b39856b077604accdb57570ead8b335/packages/Discretization/src/DTK_FE.hpp#L247

Reply:
OK.  Sorry I should have realized you have a separate commit for that one.  Good!